tests/assembly/62531-array-cmp.rs

```diff
@@ -0,0 +1,21 @@
+// Ensure the asm for array comparisons is properly optimized on x86.
+
+//@ min-llvm-version: 12
```

Comment:
Drop this line -- no need to specify min-llvm-version if it's lower than the oldest supported.